### PR TITLE
#82 spanish translation of index, archive, contact, manifiesto, offering, projects and work

### PR DESCRIPTION
**Pull request by albertofountain-aircury**
[Website Texts- Spanish (1).docx](https://github.com/curacultura/curacultura-website/files/15148136/Website.Texts-.Spanish.1.docx)

Spanish translation

### CODE REVIEW

File: _data/content/projects/es.yml [hunk 1]
```diff
@@ -12,64 +12,192 @@ page_data:
...
+        Elisa fue invitada por la Fundación Mainel a presentar y participar
+        en un panel de líderes del sector que discutieron el papel de las
+        tecnologías y experiencias en la realización de exposiciones. Se
+        centró en varias tipologías en las que ha trabajado, desde las
+        exitosas exposiciones inmersivas del Museo V&A de Londres
+        (David Bowie Is, 'You Say You Want a Revolution?' y Pink Floyd:
+        Their Mortal Remains), Exposiciones Mundiales (particularmente
+        la Dubai Expo 2020) y experiencias de base tecnológica. El ciclo
+        de seminarios de Cultura Visual Contemporánea forma parte de
+        la oferta pública de la Mainel que se centra en la mejora de la
```
Comment (juanjoseortega-aircury) [inline _data/content/projects/es.yml:125]: "la oferta publica de la Mainel" sounds weird, maybe remove "la" before "Mainel"

File: es/index.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+layout: about
+# multilingual page pair id, this must pair with translations of this page. (This name must be unique)
+lng_pair: id_about
+
+# image for page specific usage
+img: ":home-heading.jpg"
+# publish date (used for seo)
```
Comment (cristobaldiaz-aircury) [inline es/index.md:8]: Remove commented code

File: _data/content/projects/es.yml [hunk 3]
```diff
@@ -12,64 +12,192 @@ page_data:
...
+        sociedad a través de la solidaridad y la cultura como
+        herramientas transformadoras, para conseguir que las personas
+        sean protagonistas de su propio desarrollo. En su presentación,
+        Elisa mostró cómo las experiencias inmersivas son clave para la
+        democratización de la cultura, ya que convierten a los visitantes
+        pasivos en participantes activos, cuya comprensión y
+        sentimientos son tan importantes como el conocimiento que
+        comparten los curadores y las instituciones mismos. Tanto los
```
Comment (juanjoseortega-aircury) [inline _data/content/projects/es.yml:133]: "las instituciones mismos" it's wrong, it should be "mismas"

File: es/tabs/archive.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+---
```
Comment (cristobaldiaz-aircury) [inline es/tabs/archive.md:1]: Any reason to have this file?

**Review by cristobaldiaz-aircury — CHANGES_REQUESTED** [review]

File: _data/content/projects/es.yml [hunk 5]
```diff
@@ -12,64 +12,192 @@ page_data:
...
       post: |
-        # Rise-Rosa-Rage Archivo de Gráficos Sociopolíticos, Roma, Italia / Online
+        Rise-Rosa-Rage es un archivo de gráfica sociopolítica que
+        actualmente consta de más de 4.000 objetos físicos, digitales y en
+        la biblioteca de referencia. CuraCultura es responsable de la
+        curación, el desarrollo de colecciones y la gestión del archivo, así
+        como de la catalogación en línea, la traducción de contenidos, la
+        redacción de las cartelas de los objetos y de artículos, y el
+        desarrollo web. Las colecciones datan predominantemente de las
+        décadas de los 1960 hasta los 1990, con algunas piezas también
```
Comment (juanjoseortega-aircury) [inline _data/content/projects/es.yml:153]: In English translation we have 1960 to 1980, and here in Spanish we have it from 1960 to 1990. maybe this should be asked to the client

File: _data/content/projects/es.yml [hunk 6]
```diff
@@ -12,64 +12,192 @@ page_data:
...
+        hasta el día de hoy. Provienen de o tienen contenido relacionado
+        con más de 60 territorios diferentes, y están representados en más
+        de 20 idiomas. Sus temas clave son la solidaridad, la reivindicación
+        de derechos, la memoria y las lucha por la paz y la autonomía. En
```
Comment (juanjoseortega-aircury) [inline _data/content/projects/es.yml:157]: "las lucha" should be at singular, so should be "la"

File: es/index.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+      nuestra <a class="link" href="tabs/offering.html">página de Oferta</a>.</p>
+</div>
+<div>
+    <div class="photoTitle">
+        <img class="imgTitles" src="/assets/img/default/logo.webp" alt="Logo image">
+        <h2 class="title2"> Filosofía  </h2>
+    </div>
+       <p>"Curar es cuidar”. Cuidar la cultura significa cuidar a las personas. Esta filosofía nos guía tanto en los proyectos que creamos o los que aceptamos, como también en nuestro comportamiento como individuos y como empresa, hacia nuestros colaboradores, clientes, compañeros y el público en general. Luchamos por una sociedad y un sector cultural más cálido y más empático para todos. Actualmente estamos elaborando un Manifiesto sobre lo que esto significa en la práctica y, como cuidar también significa escuchar y compartir, nos gustaría invitarte a contribuir con tus ideas a través de nuestra página  <a class="link">Un Manifiesto Colaborativo para Curar con Cuidado</a>. Puede ver todos los puntos recopilados hasta la fecha en este <a class="link">documento de Google no editable</a>, donde el suyo se agregará después de la revisión para garantizar que mantengamos un espacio seguro.</p>
```
Comment (juanjoseortega-aircury) [inline es/index.md:25]: "como cuidar" should have accent at "como"

File: es/tabs/manifiesto.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+<div class="padding-container even-background">
+    <div>
+        <div class="photoTitle">
+            <img class="imgTitles" src="/assets/img/default/logo.webp" alt="Logo image">
+            <h2 class="title2">Un enfoque colaborativo para hacer que la cultura sea más amable</h2>
+        </div>
+        <p>CuraCultura no es sólo una empresa. También es una filosofía, definida por su propio nombre:
+          cuidamos de las personas a través de la cultura, y cuidamos la cultura por el bien de las
+          personas. Nos propusimos hacer del sector cultural un lugar más amable para todos los que lo
+          habitan, ya sean inquilinos permanentes o visitantes poco frecuentes, trabajadores y
```
Comment (juanjoseortega-aircury) [inline es/tabs/manifiesto.md:14]: "inquilinos" here sounds weird for me, maybe this should be reviewed or asked to the client

**Review by juanjoseortega-aircury — COMMENTED** [review]
Typo error's from client document

File: es/tabs/archive.md (hunk 4 above)
Comment (cristobaldiaz-aircury) [reply es/tabs/archive.md:1]: If we're not using it in English it should be removed then.

File: _data/lang/es.yml [hunk 9]
```diff
@@ -38,7 +38,7 @@ navigation:
 color_scheme_switch_tooltip: Apariencia
 
 search:
-  placeholder: Búsqueda
+  placeholder: Buscar
   noResultText: No se ha encontrado ningún resultado.
 
 image_viewer:
```
Comment (juanjoseortega-aircury) [inline _data/lang/es.yml:44]: At line 51 `some_rights_reserved` should me modified to Spanish translation

File: es/tabs/offering.md [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+                            <li>Escritura de textos y guiones (gráficos y medios)</li>
+                            <li>Traducción</li>
+                        </ul>
+                    </div>
+                </div>
+                <p>Nuestro equipo de personas fuertes, empáticas y amables hablan su idioma:</p>
+                <div class="companiesProjectsContainer">
+                        <ul class="sub-item">
+                            <li>Todos somos como mínimo bilingües en:</li>
+                            <li>inglés</li>
```
Comment (juanjoseortega-aircury) [inline es/tabs/offering.md:76]: Languages should be at uppercase

File: es/tabs/work.md [hunk 11]
```diff
@@ -0,0 +1,178 @@
...
+                Oxford, Reino Unido, donde trabajó como curadora adjunta y
+                registradora en varios museos, y así conoció a Elisa. Juntos
+                colaboraron en un proyecto de investigación sobre el monumento de
+                Buzludzha en Bulgaria entre 2014 y 2016, del que surgieron un libro,
+                artículos y una exposición. Caroline tiene una licenciatura en
+                Historia, Historia del Arte y Geografía de la Sorbona, París, y tres
+                maestrías: en Arte Moderno e Inglés de la Universidad Oxford
+                Brookes, en Estudios de Museos de la Universidad de Leicester y en
+                Teoría del Arte Contemporáneo de Goldsmiths, Universidad de
+                Londres. (todo el Reino Unido).
```
Comment (cristobaldiaz-aircury) [inline es/tabs/work.md:105]: Remove the stop in `Universidad de Londres. (todo el Reino Unido).`. It should look like this:
``Universidad de Londres (todo el Reino Unido).``

File: _data/content/projects/es.yml [hunk 12]
```diff
@@ -12,64 +12,192 @@ page_data:
...
+      date: "Desde septiembre 2022"
       post: |
-        # Módulo del Curso MA Escuela Superior de Arte y Diseño de Barreira, Valencia, España
+        La edición inaugural del Master en Mercados Globales del Arte
+        de la Escuela Superior de Arte y Diseño Barreira tuvo lugar en el
+        año académico 2022-23, cuando CuraCultura fue invitada por el
+        Director para planificar e impartir el módulo sobre los escenarios
+        culturales y creativos del ‘Medio Oriente’ y África del Norte. Luego
+        de una exitosa primera ejecución, el programa continuará
+        ejecutándose anualmente. Partiendo de una exploración de las
```
Comment (cristobaldiaz-aircury) [inline _data/content/projects/es.yml:186]: Maybe `realizándose` instead of `ejecutándose`? Ask client

**Review by cristobaldiaz-aircury — CHANGES_REQUESTED** [review]

**Review by cristobaldiaz-aircury — APPROVED** [review]

**Pull request merged**